client/app/scripts/components/app.js

```diff
@@ -197,7 +196,6 @@ class App extends React.Component {
     } = this.props;
 
     const className = classNames('scope-app', { 'time-travel-open': timeTravelSupported });
-    const isIframe = window !== window.top;
```

Comment:
why this is removed?

Reply:
it is checking the condition for the iframe, if topology open in the iframe it will it will not show the logo, else it will show the logo